packages/admin/config/filament.php

```diff
@@ -207,10 +207,12 @@
         ],
         'sidebar' => [
             'is_collapsible_on_desktop' => false,
+            'has_hamburger_icon_on_desktop' => false,
```

Comment:
So this config key will probably become something like:

```suggestion
            'can_be_hidden_on_desktop' => false,
```